instant-booking/templates/booking-confirmed-customer/booking-confirmed-customer-html.html

```diff
@@ -0,0 +1,391 @@
+    </tr>
+  </tbody>
+</table>
+
+{{else}}
+<table align="center" border="0" cellPadding="0" cellSpacing="0" role="presentation" width="100%">
+  <tbody>
+    <tr>
+      <td>
+      <td>
+        <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">
+          {{> format-money money=unit-price}} ×
+          {{number quantity}}
+          {{inflect quantity "day" "days"}}</p>
+      </td>
+      <td style="text-align:right">
+        <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">
+          {{> format-money money=line-total}}</p>
+      </td>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+{{/if}}
+{{/eq}}
+{{#eq "line-item/night" code}}
+{{#if seats}}
+<table align="center" border="0" cellPadding="0" cellSpacing="0" role="presentation" width="100%">
+  <tbody>
+    <tr>
+      <td>
+      <td>
+        <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">
+          {{> format-money money=unit-price}} ×
+          {{number units}}
+          {{inflect units "night" "nights"}}</p>
+        <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">Seats ×
+          {{number seats}}</p>
+      </td>
+      <td style="text-align:right">
+        <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">
+          {{> format-money money=line-total}}</p>
+      </td>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+{{else}}
+<table align="center" border="0" cellPadding="0" cellSpacing="0" role="presentation" width="100%">
+  <tbody>
+    <tr>
+      <td>
+      <td>
+        <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">
+          {{> format-money money=unit-price}} ×
+          {{number quantity}}
+          {{inflect quantity "night" "nights"}}</p>
+      </td>
+      <td style="text-align:right">
+        <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848">
+          {{> format-money money=line-total}}</p>
+      </td>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+{{/if}}
+{{/eq}}
+{{/contains}}
+{{/each}}
+<hr
+  style="width:100%;border:none;border-top:1px solid #eaeaea;border-color:#E1E1E1;margin:20px 0;margin-left:-24px;padding-right:48px" />
+<table align="center" border="0" cellPadding="0" cellSpacing="0" role="presentation" width="100%">
+  <tbody>
+    <tr>
+      <td>
+      <td>
+        <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848;margin-top:1px;font-weight:700">Total price
+        </p>
+      </td>
+      <td style="text-align:right">
+        <p style="font-size:16px;line-height:1.4;margin:16px 0;color:#484848;margin-top:1px;font-weight:700">
+          {{> format-money money=payin-total}}</p>
+      </td>
+      </td>
+    </tr>
+  </tbody>
+</table>
+</td>
+</tr>
+</tbody>
+</table>
+<table style="padding:16px 0 0" align="center" border="0" cellPadding="0" cellSpacing="0" role="presentation"
+  width="100%">
+  <tbody>
+    <tr>
+      <td><a href="
+{{marketplace.url}}/order/
```

Comment:
Line breaks in the URL mess up the redirect to the template, so they'd need to be checked and fixed